tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -480,10 +509,32 @@ export class WebGPUBackend extends KernelBackend {
     if (shouldTimeProgram) {
       query = this.endTimer(query);
       this.activeTimers.push(
-          {name: program.constructor.name, query: this.getQueryTime(query)});
+          {name: program.constructor.name, query: this.getQueryTime(this.cc -2)});
     }
     return output as {} as K;
   }
+  async getResult(counter: number) {
+    let buf = this.dstBuffer;
+    await util.repeatedTry(() => {
+      return (this.cc === this.nn * 2);
+    });
```

Comment:
When to reset this.cc to 0 so that you can begin anther round of profiling?

Reply:
Done.